### ISSUE TITLE
Pedestal http listens on localhost by default

### ISSUE THREAD

**Issue by steveblamey** [wardle/clods#9]
Pedestal configures the http server to listen on `localhost` by default. When running the REST server inside a container the http server needs to listen on `0.0.0.0` to ensure requests are routed to the service.

The host can be defined in the service map as `::http/host`, according to the documentation. It would be handy if the host value was a runtime option.

**Comment by wardle** [comment]
Thanks @steveblamey  - good idea. I do the same in Hermes as well as configuring allowed origins as well for CORS policies.

**Comment by wardle** [comment]
Of course, in meantime, it is possible to work around this issue in containers by using something like caddy or nginx. I use caddy to make use of TLS, for example, and get automatic updating of certificates etc. I simply run a local HTTP server for one of my services, and then use caddy to reverse proxy.
